charts/gateway/README.md

```diff
@@ -678,6 +697,41 @@ ingress:
 | `pmtagger.podSecurityContext`    | [Pod Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)              | `[]` |
 | `pmtagger.containerSecurityContext`    | [Container Security Context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container)          | `{}` |
 
+### Redis Configuration
+This enables integration with [Redis](https://redis.io/). The following sections configure a redis configuration file on the Gateway. The following properties in config.systemProperties will need to be updated
+
+Comment out the following
+```
+# com.l7tech.server.extension.sharedKeyValueStoreProvider=embeddedhazelcast
+# com.l7tech.server.extension.sharedCounterProvider=ssgdb
+```
+Uncomment the following
+```
+# com.l7tech.server.extension.sharedKeyValueStoreProvider=redis
+# com.l7tech.server.extension.sharedCounterProvider=redis
+# com.l7tech.server.extension.sharedRateLimiterProvider=redis
+```
+
+| Parameter                        | Description                               | Default                                                      |
+| -----------------------------    | -----------------------------------       | -----------------------------------------------------------  |
+| `config.redis.enabled`          | Enable redis configuration | `false`  |
+| `config.redis.existingConfigSecret`          | Use an existing config secret - must contain a key called redis.properties | `false`  |
+| `config.redis.subChart.enabled`          | Deploy the redis subChart | `true`  |
+| `config.redis.groupName`          | Redis Group name | `l7GW`  |
+| `config.redis.auth.enabled`          | Use auth for Redis | `false`  |
+| `config.redis.auth.username`          | Redis username | ``  |
+| `config.redis.auth.password.encoded`          | Password is encoded | `false`  |
+| `config.redis.auth.password.value`          | Redis password | `mypassword`  |
+| `config.redis.sentinel.enabled`                | Enable sentinel configuration   | `true` |
+| `config.redis.sentinel.masterSet`          | Redis Master set | `mymaster`  |
+| `config.redis.sentinel.nodes`          | Array of sentinel nodes and ports | `[]`  |
+| `config.redis.standalone.host`                | Redis host if sentinel is not enabled   | `redis-standalone` |
+| `config.redis.standalone.port`                | Redis port if sentinel is not enabled   | `6379` |
+| `config.redis.tls.enabled`    | Enable SSL/TLS              | `false` |
+| `config.redis.tls.existingSecret`    | Use an existing secret - must contain a key called tls.crt        | `` |
+| `config.redis.tls.verifyPeer`    | Verify Peer             | `false` |
```

Comment:
We might want default verifyPeer to true

Reply:
verifyPeer now defaults to true